### PR TITLE
Improved Jarnovanleeuwen amendments

### PR DESCRIPTION
**Pull request by monofone**
Added a test and a hint to the README to line out the usage

### CODE REVIEW

File: tests/Unit/Digitick/Sepa/TransferInformation/CustomerDirectDebitTransferInformationTest.php [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<?php
+namespace tests\Unit\Digitick\Sepa\TransferInformation;
+
+/***************************************************************
+ *  (c) 2015 networkteam GmbH - all rights reserved
```
Comment (BenjaminPaap) [inline tests/Unit/Digitick/Sepa/TransferInformation/CustomerDirectDebitTransferInformationTest.php]: @jarnovanleeuwen I'm not that familiar with all those licenses but I think this conflicts with our `GNU Lesser General Public License v3.0` which we used for this project. Would you mind removing this?

Comment (BenjaminPaap) [inline tests/Unit/Digitick/Sepa/TransferInformation/CustomerDirectDebitTransferInformationTest.php]: Sorry, this seems to be introduced by @monofone

Comment (monofone) [inline tests/Unit/Digitick/Sepa/TransferInformation/CustomerDirectDebitTransferInformationTest.php]: Ah, damn not a good Moment to create PRs being at symfonyCon. I'll fix it tomorrow

Comment (BenjaminPaap) [inline tests/Unit/Digitick/Sepa/TransferInformation/CustomerDirectDebitTransferInformationTest.php]: :+1: and have fun :beers:

**Comment by jarnovanleeuwen** [comment]
Thank you for adding a test @monofone. Have fun at symfonyCon!

**Pull request merged**